README.md

```diff
@@ -40,7 +40,7 @@ Let’s see how we can use aixplain to run a machine translation model.
 
 ```python
 from aixplain.factories.model_factory  import ModelFactory
-model = ModelFactory.get("61b27086c45ecd3c10d0608c") # Got the ID of an MT model from on our platform
+model = ModelFactory.get("61dc52976eb5634cf06e97cc") # Get the ID of a model from our platform. This example shows an English to French translation model.
```

Comment:
Would it make sense to add the link to the model in the platform so the users could see more info?
(e.g. https://platform.aixplain.com/discovery/model/61dc52976eb5634cf06e97cc)